crates/sui-framework/packages/sui-framework/sources/crypto/bls12381.move

```diff
@@ -37,6 +37,7 @@ public struct Scalar {}
 public struct G1 {}
 public struct G2 {}
 public struct GT {}
+public struct G1Uncompressed {}
```

Comment:
pls add some documentation in the comment below about the representation, and that it's meant to be internal